bluesky/run_engine.py

```diff
@@ -251,23 +251,31 @@ def __init__(self, md=None, *, md_validator=None):
             'wait_for': self._wait_for,
         }
 
-        # public dispatcher for callbacks processed on the main thread
+        # public dispatcher for callbacks
+        # The Dispatcher's public methods are exposed through the
+        # RunEngine for user convenience.
         self.dispatcher = Dispatcher()
         self.ignore_callback_exceptions = True
         self.event_timeout = 0.1
         self.subscribe = self.dispatcher.subscribe
         self.unsubscribe = self.dispatcher.unsubscribe
 
-        # private registry of "critical" callbacks
+        # private dispatcher of critical callbacks
         # which get a lossless Event stream and abort a run if they raise
-        self._lossless_cb_registry = CallbackRegistry(
-            allowed_sigs=DocumentNames)
+        # Ths subscribe/unsubscribe are exposed through the RunEngine
+        # as subscribe_lossless and unsubscribe_lossless, defined below
+        # to provide special docstrings.
+        self._lossless_dispatcher = Dispatcher()
 
         self.verbose = False
 
         loop.call_soon(self._check_for_trouble)
         loop.call_soon(self._check_for_signals)
 
+        # aliases for back-compatibility
+        self._unsubscribe_lossless = self.unsubscribe_lossless
```

Comment:
These should be done at class level as `_unsubscribe_lossless = unsubscribe_lossless`